pkgs/applications/networking/cluster/eks-node-viewer/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "eks-node-viewer";
+  version = "6a915ff6c89b00d5e8ff90e3fe0ce65dd12eca17";
```

Please move the hash to rev a d apply the following

```suggestion
  version = "2022-12-10";
```